pwndbg/gdblib/kernel/kallsyms.py

```diff
@@ -112,7 +123,7 @@ def find_token_table(self) -> int:
             if len(ascii_candidates) == 1:
                 candidates = ascii_candidates
             elif len(candidates) == 0:
-                print(M.error("No candidates for token_table"))
+                # print(M.error("No candidates for token_table, maybe uncompressed kallsyms"))
```

Comment:
Since you will print if you detect uncompressed elsewhere already, we probably won't need this print, and it's commented out already.